WMF Framework/Remote Notifications/Model/RemoteNotificationsModelController.swift

```diff
@@ -285,37 +258,4 @@ final class RemoteNotificationsModelController: NSObject {
             self.save()
         }
     }
-
-    // MARK: Notifications
-
-    @objc private func managedObjectContextDidSave(_ note: Notification) {
-        guard let userInfo = note.userInfo else {
-            assertionFailure("Expected note with userInfo dictionary")
-            return
-        }
-        if let insertedObjects = userInfo[NSInsertedObjectsKey] as? Set<NSManagedObject>, !insertedObjects.isEmpty {
-            postModelDidChangeNotification(ofType: .addedNewNotifications, withNotificationsFromObjects: insertedObjects)
-        }
-        if let updatedObjects = userInfo[NSUpdatedObjectsKey] as? Set<NSManagedObject>, !updatedObjects.isEmpty {
-            postModelDidChangeNotification(ofType: .updatedExistingNotifications, withNotificationsFromObjects: updatedObjects)
-        }
-    }
-
-    private func postModelDidChangeNotification(ofType modelChangeType: RemoteNotificationsModelChangeType, withNotificationsFromObjects objects: Set<NSManagedObject>) {
-        let notifications = objects.compactMap { $0 as? RemoteNotification }.filter { $0.state == nil }
-        guard !notifications.isEmpty else {
-            return
-        }
-        let notificationsGroupedByCategoryNumber = Dictionary(grouping: notifications, by: { NSNumber(value: $0.category.rawValue) })
-        let modelChange = RemoteNotificationsModelChange(type: modelChangeType, notificationsGroupedByCategoryNumber: notificationsGroupedByCategoryNumber)
-        let responseCoordinator = RemoteNotificationsModelChangeResponseCoordinator(modelChange: modelChange, modelController: self)
-        DispatchQueue.main.async {
-            NotificationCenter.default.post(name: RemoteNotificationsModelController.modelDidChangeNotification, object: responseCoordinator)
```

Comment:
Apologies if I'm missing its usage in the subsequent PRs, but would it also make sense to remove line 45 which no longer appears to be used?
https://github.com/wikimedia/wikipedia-ios/blob/1c0ee404f482bc95e05d5fec7d19ae4c9b01b918/WMF%20Framework/Remote%20Notifications/Model/RemoteNotificationsModelController.swift#L45

Reply:
Nice catch! I pushed a commit removing this.